pkgs/by-name/pe/perl-debug-adapter/package.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, buildNpmPackage
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, perl
+# Needed if you want to use it for a perl script with dependencies.
+, extraPerlPackages ? []
+}:
+
+let
+  perlInterpreter = perl.withPackages(ps: [
+    ps.PadWalker
+  ] ++ extraPerlPackages);
+in buildNpmPackage rec {
+  pname = "perl-debug-adapter";
+  version = "1.0.6";
+
+  src = fetchFromGitHub {
+    owner = "Nihilus118";
+    repo = "perl-debug-adapter";
+    rev = version;
+    hash = "sha256-IXXKhk4rzsWSPA0RT0L3CZuKlgTWtweZ4dQtruTigRs=";
+  };
+
+  npmDepsHash = "sha256-iw7+YC4qkrTVEJuZ9lnjNlUopTCp+fMNoIjFLutmrMw=";
+
+  npmBuildScript = "compile";
+
+  makeWrapperArgs = [
+    "--prefix" "PATH" ":" "${perlInterpreter}/bin"
+  ];
+  passthru = {
+    inherit perlInterpreter;
+  };
+
+  meta = {
+    # Upstream doesn't provide an official 1 line description, so this is a
+    # certain paraphrase over one of the sentences in the README.
+    description = "Debug adapter, invokes perl -d and handles communication with VS Code or other editors";
+    inherit (src.meta) homepage;
+    changelog = "${src.meta.homepage}/blob/${version}/CHANGELOG.md";
```

```suggestion
    changelog = "https://github.com/Nihilus118/perl-debug-adapter/blob/${version}/CHANGELOG.md";
```
you can't click or copy that